README.md

```diff
@@ -6,10 +8,77 @@
 |**Frontend Coverage**| [![Codecov-Frontend-Dev](https://codecov.io/gh/raft-tech/TANF-app/branch/raft-tdp-main/graph/badge.svg?flag=dev-frontend)](https://codecov.io/gh/raft-tech/TANF-app?flag=dev-frontend) | [![Codeco-Frontend-HHS](https://codecov.io/gh/HHS/TANF-app/branch/main/graph/badge.svg?flag=main-frontend)](https://codecov.io/gh/HHS/TANF-app?flag=main-frontend)   |
 |**Backend Coverage**|  [![Codecov-Backend-Dev](https://codecov.io/gh/raft-tech/TANF-app/branch/raft-tdp-main/graph/badge.svg?flag=dev-backend)](https://codecov.io/gh/raft-tech/TANF-app/branch/raft-tdp-main?flag=dev-backend)|   [![Codecov-Backend-HHS]( https://codecov.io/gh/HHS/TANF-app/branch/main/graph/badge.svg?flag=main-backend)](https://codecov.io/gh/HHS/TANF-app/branch/main?flag=main-backend) |
 
-***Due to limitations imposed by Github and occasional slow server response times, some badges may require a page refresh to load.**
+**Due to limitations imposed by Github and occasional slow server response times, some badges may require a page refresh to load.**
+
+## Office of Family Assistance (OFA) & Temporary Assistance for Needy Families (TANF) Data Reporting System.
```

Comment:
I think this line can be deleted (I'd like to clean up use of TDRS and replace with TDP throughout, unless referring to the current system)

Reply:
> I think this line can be deleted (I'd like to clean up use of TDRS and replace with TDP throughout, unless referring to the current system)

Yes, @lfrohlich! Style guide prioritized for the next time will be helpful here so we all use the same terms.

Reply:
@lfrohlich just to make sure I understand, you would like to delete the entire line 13?

Reply:
I have updated the line to say (TANF) Data Portal

Reply:
Thanks. I was thinking we could delete it but this works too